packages/config/src/models/question.ts

```diff
@@ -132,6 +158,11 @@ const questionSchema = new Schema({
     type: [conditionals],
     default: undefined
   },
+  validator: [
+    {
+      type: validator
+    }
+  ],
```

Comment:
@rikukissa @tahmidrahman-dsi @Zangetsu101 would we need a migration so that questions created in OpenCRVS v1.3 would work in v1.4?  From the look of these model changes, I dont think so.  Do you agree?

Reply:
The fields look to be optional. So should just work

Reply:
👍 that's correct